azurelinuxagent/common/osutil/bigip.py

```diff
@@ -211,8 +199,8 @@ def del_account(self, username):
         :param username:
         :return:
         """
-        shellutil.run("> /var/run/utmp")
```

Comment:
This command was just using redirection to create empty files, replaced it with `touch` command